GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
I don't think I fully understand this function. The context should be available irrespective of the hierarchy. It is the GVRActivity that calls setScript. The script in turn could be used to get the context. Do  you have an example of where this is used in a subclass? maybe that would give a better idea.